cluster/manifests/kube-dns/node-local-svc.yaml

```diff
@@ -0,0 +1,23 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: kube-dns-local
+  namespace: kube-system
+  labels:
+    application: kube-dns-local
+    kubernetes.io/cluster-service: "true"
+    kubernetes.io/name: "KubeDNS"
+  annotations:
+    prometheus.io/path: /metrics
+    prometheus.io/port: "9054"
+    prometheus.io/scrape: "true"
+spec:
+  selector:
+    application: dnsmasq-node
+  type: NodePort
+  externalTrafficPolicy: Local
```

Comment:
I guess we can just use `ClusterIP` and drop the `externalTrafficPolicy` as this service is only needed for the metrics collection now.

Reply:
no ClusterIP is not possible because of the port range

Reply:
But afaict we don't need the node ports anymore, since we use hostNetwork on the pods?

Reply:
Is this hurting you?
From my side I don't see a value in changing and as far as I remember there was a port assign problem.

Reply:
I just think it makes sense to remove unnecessary configuration which is not used. It's confusing to have `externalTrafficPolicy: Local` explicitly defined if we are not even using it. The service is only used for gathering metrics, so it should be as simple as possible IMO.

Reply:
I will change it

Reply:
>  it was our hack with the 10.4.0.1 I think why we had this service nodeport.

yes, that I was I meant :)